charts/incubator/tt-rss/values.yaml

```diff
@@ -0,0 +1,181 @@
+    runAsNonRoot: false
+    runAsUser: 0
+    runAsGroup: 0
+persistence:
+  html:
+      enabled: true
+      targetSelector:
+        main:
+          main:
+            mountPath: /var/www/html
+        nginx:
+          nginx:
+            mountPath: /var/www/html
+            readOnly: true
+        updater:
+          updater:
+            mountPath: /var/www/html
+  config:
+    enabled: true
+    targetSelector:
+        main:
+          main:
+            mountPath: /opt/tt-rss/config.d
+        updater:
+          updater:
+            mountPath: /opt/tt-rss/config.d
+  # plugins:
+  #   enabled: true
+  #   mountPath: /var/www/html/tt-rss/plugins.local
+  # themes:
+  #   enabled: true
+  #   mountPath: /var/www/html/tt-rss/themes.local
+cnpg:
+  main:
+    enabled: true
+    user: tt-rss
+    database: tt-rss
+portal:
+  open:
+    enabled: true
+workload:
+  main:
+    podSpec:
+      containers:
+        main:
+          probes:
+            liveness:
+              type: exec
+              command:
+                - /bin/sh
+                - -c
+                - netstat -npl | grep :9000
+            readiness:
+              type: exec
+              command:
+                - /bin/sh
+                - -c
+                - ls /var/www/html/tt-rss/.app_is_ready
+            startup:
+              type: tcp
+              port: "{{ .Values.service.app.ports.app.port }}"
+          env:
+            TTRSS_SELF_URL_PATH: ""
+            # tt-rss tests with "-z" so an empty string should still allow the randomly generated password to be made.
+            ADMIN_USER_PASS: ""
+            TTRSS_DB_NAME: "{{ .Values.cnpg.main.database }}"
+            TTRSS_DB_USER: "{{ .Values.cnpg.main.user }}"
+            TTRSS_DB_PORT: "5432"
+            TTRSS_DB_PASS:
+              secretKeyRef:
+                name: cnpg-main-user
+                key: password
+            TTRSS_DB_HOST:
+              secretKeyRef:
+                name: cnpg-main-urls
+                key: host
+  nginx:
+    enabled: true
+    type: Deployment
+    podSpec:
+      containers:
+        nginx:
+          enabled: true
+          primary: true
+          imageSelector: nginxImage
+          probes:
+            readiness:
+              enabled: true
+              type: http
+              path: "/public.php?op=healthcheck"
+              port: "{{ .Values.service.main.ports.main.targetPort }}"
+            liveness:
+              enabled: true
+              type: http
+              path: "/public.php?op=healthcheck"
+              port: "{{ .Values.service.main.ports.main.targetPort }}"
+            startup:
+              enabled: true
+              type: tcp
+              port: "{{ .Values.service.main.ports.main.targetPort }}"
+          env:
+            APP_UPSTREAM: '{{ include "tc.v1.common.lib.chart.names.fullname" . }}-app.{{ .Release.Namespace }}.svc.cluster.local'
+            RESOLVER: "kube-dns.kube-system.svc.cluster.local"
```

Comment:
With the app_upstream changed, try to comment out this and see if it can resolve.